etc/systemd/system/zfs-import.target.in

```diff
@@ -0,0 +1,6 @@
+[Unit]
+Description=ZFS pool import target
+
+[Install]
+WantedBy=zfs-mount.service
```

Comment:
IMHO also redundant, `zfs-mount.service` already has an `After=`, and the main `zfs.target` already ensures that it is enabled by default

Reply:
If I understand systemd correctly, I think that as long as zfs.target is enabled, this `WantedBy=zfs-mount.service` is redundant. I suppose I was worried someone might enable `zfs-mount` and not `zfs.target`. I can remove that extra dependency.

Reply:
yes, but we already have a default chain of
`multi-user.target` wants `zfs.target` wants `zfs-FOO.target/service` that makes sure all the services and targets are enabled. so we only need ordering between mount and import, if both are enabled.

`zfs-mount.service` works independently of `zfs-import.target` and the `zfs-import-FOO` services (e.g., if your rpool has been imported but only the bootfs mounted in initramfs/dracut), it should just run after them if they are also enabled.

it's just a minor nitpick though, and the old units had that WantedBy as well, so feel free to ignore this ;)  the same would IMHO also apply to `zfs-share` -> `zfs-mount`, which is entirely unrelated to this PR

Reply:
This would be a good time to cleanup the other unit dependencies that don't make sense.  Although it should be done with two patches so we have the option of cherry-picking the minimal change back to the 0.7 releases.